src/styles/blocks/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, sans-serif;
+  background-color: #fff;
+  color: #000;
+
+  &__img {
+    object-fit: contain;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $secondary-color;
+  }
+
+  &__reviews-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
```

use nesting, fix in all cases
```suggestion


  &__reviews {
    font-weight: 400;
    font-size: 10px;
    line-height: 14px;
    color: $main-color;
    
    &-wrapper {
      display: flex;
      justify-content: space-between;
      align-items: end;
      margin-bottom: 24px;
    }
  }
```